src/FsCheck/Runner.fs

```diff
@@ -59,10 +63,11 @@ type Config =
       ///overridden)
       Arbitrary     : list<Type>
       ///A custom test runner, e.g. to integrate with a test framework like xUnit or NUnit. 
-      Runner        : IRunner }
+      Runner        : IRunner
+      ///If set, inputs for property generation and property evaluation will be runned in parallel. 
+      ParallelRunConfig : ParallelRunConfig option}
```

Comment:
> I guess the option type also encodes whether or not to do a parallel run

Right.
Regarding `MaxDegreeOfParallelism` I see only two reasonable options: `Environment.ProcessorCount` - for cpu bound crunching and 1 - for async i/o bound work (however `Environment.ProcessorCount` is also ok for i/o). Overall this parameter may confuse users, but I don't want to left them with hardcoded defaults. So I'm going to put dense comments in code and suggest to give a couple of lines regarding this in documentation (I can contribute to it by my own).
If you have different vision on configuration style - I'm ok to stick with yours.